mirgecom/eos.py

```diff
@@ -79,6 +79,7 @@ class GasEOS:
     .. automethod:: gamma
     .. automethod:: transport_model
     .. automethod:: get_internal_energy
+    .. automethod:: get_species_enthalpies
```

Comment:
For at least one of them, it was because `internal_energy` was already taken, and the `get_internal_energy` version is an inversion routine with a different interface.  But mostly it was to match the underlying interface that the EOS is calling.

Reply:
Whoah, I totally missed that we had both `internal_energy` and `get_internal_energy`. I shouldn't have let that pass review. 🙂 I guess it's not _so_ bad since they take different arguments, but there's a probably a more descriptive name we could use for `get_internal_energy`.

Could we remove the `get_` prefixes for the new methods here?

Reply:
(fabd833) and #485